src/modules/mc_att_control/AttitudeControl/AttitudeControl.cpp

```diff
@@ -68,17 +68,21 @@ matrix::Vector3f AttitudeControl::update(const Quatf &q) const
 		qd_red = qd;
 
 	} else {
-		// transform rotation from current to desired thrust vector into a world frame reduced desired attitude
+		// Transform rotation from current to desired thrust vector into a world frame reduced desired attitude.
+		// This is a right multiplication as the tilt error quaternion obtained from two Z vectors is referenced at the origin.
```

Comment:
```suggestion
		// This is a right multiplication as the tilt error quaternion obtained from two Z vectors referenced at the origin.
```
How about "... from unit Z-axis vectors of body and world frame."?
" Z vectors referenced at the origin" is correct but would confuse me.

Reply:
I wanted to say that the "tilt error quaternion" is expressed in the world frame (and yes, that's because both Z vectors are expressed in the world frame)

See:
![image](https://github.com/user-attachments/assets/9085c394-d7b0-4f17-9c9f-5d4e8f5c6b94)
Note that the "local frame" in this paper is our "body frame"

Reply:
@MaEtUgR I reworded it slightly